src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -169,51 +130,6 @@ def privacy_experience_detail(
 
     # Temporarily stash the privacy notices on the experience for display
     experience.privacy_notices = experience.get_related_privacy_notices(
-        db, region, show_disabled
+        db, show_disabled
     )
     return experience
-
-
-def ensure_unique_ids(
-    privacy_experience_updates: List[PrivacyExperienceWithId],
-) -> None:
-    """Verifies privacy experience ids are unique in request to avoid unexpected behavior"""
-    request_ids: List[str] = [update.id for update in privacy_experience_updates]
-    if len(request_ids) != len(set(request_ids)):
-        raise HTTPException(
-            HTTP_422_UNPROCESSABLE_ENTITY,
-            detail="Duplicate privacy experience ids submitted in request.",
-        )
-
-
-@router.patch(
-    urls.PRIVACY_EXPERIENCE,
-    status_code=HTTP_200_OK,
-    response_model=List[PrivacyExperienceResponse],
-    dependencies=[
-        Security(verify_oauth_client, scopes=[scope_registry.PRIVACY_EXPERIENCE_UPDATE])
-    ],
-)
-def privacy_experience_bulk_update(
-    *,
-    db: Session = Depends(deps.get_db),
-    privacy_experience_updates: conlist(PrivacyExperienceWithId, max_items=50),  # type: ignore
-) -> List[PrivacyExperience]:
-    """
-    Bulk update privacy experiences.  Related notices are returned in the response.
-    """
-    ensure_unique_ids(privacy_experience_updates)
-
-    loaded_privacy_experiences: List[PrivacyExperience] = [
-        get_privacy_experience_or_error(db, experience_update.id)
-        for experience_update in privacy_experience_updates
-    ]
-
-    return [
-        existing_experience.update(
-            db, data=experience_update_data.dict(exclude_unset=True)
-        )
-        for (existing_experience, experience_update_data) in zip(
-            loaded_privacy_experiences, privacy_experience_updates
-        )
-    ]
```

Comment:
You no longer update privacy experiences directly.  They are indirectly updated in the backend as Privacy Notices and ExperienceConfigs change.